TAT-2019.1/EntryPoint.cs

```diff
@@ -0,0 +1,48 @@
+﻿using System;
+
+namespace TAT_2019._1
+{
+    /// <summary>
+    /// the main class Program,entry point to the application.
+    /// From here we get the string from command line, and check for compliance with conditions.
+    /// </summary>
+    ///<returns> 0,when program work successfully</returns>
+    ///<returns> 1,when string, from command line, have less than 2 symbols</returns>
+    ///<returns> 2,when program have unpredictable errors or warnings</returns>
+    class EntryPoint
+    {
+        /// <summary>
+        /// entrance to the application
+        /// </summary>
+        /// <param name="args"></param>
+        static int Main(string[] args)
+        {
+            try
+            {
+                if (args[0].Length < 2 && args.Length < 2)
```

Comment:
Why do you need more than one argument? Is it better to check that there is an argument?